mne/viz/tests/test_3d.py

```diff
@@ -717,14 +717,14 @@ def test_plot_sensors_connectivity(renderer):
     n_channels = len(picks)
     con = np.random.RandomState(42).randn(n_channels, n_channels)
     info = raw.info
-    with pytest.raises(TypeError):
-        plot_sensors_connectivity(info='foo', con=con,
-                                  picks=picks)
-    with pytest.raises(ValueError):
-        plot_sensors_connectivity(info=info, con=con[::2, ::2],
-                                  picks=picks)
-
-    plot_sensors_connectivity(info=info, con=con, picks=picks)
+    with pytest.raises(TypeError, match='must be an instance of Info'):
+        plot_sensors_connectivity(info='foo', con=con, picks=picks)
+    with pytest.raises(ValueError, match='does not correspond to the size'):
+        plot_sensors_connectivity(info=info, con=con[::2, ::2], picks=picks)
+
+    fig = plot_sensors_connectivity(info=info, con=con, picks=picks)
+    if hasattr(fig, 'plotter'):
+        assert fig.plotter.scalar_bar.GetTitle() == 'connectivity'
```

Comment:
> Eventually resorting to the mayavi scene recording feature and click-dragging the bar yielded something like mlab_interface.children[18].children[0].children[0].children[0].scalar_bar_interactor... (reproducing from memory, I'm sure that's inaccurate) which I wasn't about to put into a test.

Indeed that's ugly but there is generally more than one path to a given vtk instance, and the recorder just chooses one based on a long line of children. But this can be simplified and relied upon since Mayavi should construct the pipeline the same way every time. For example in your test this works for Mayavi:
```suggestion
    if renderer._get_3d_backend() == 'pyvista':
        title = fig.plotter.scalar_bar.GetTitle()
    else:
        assert renderer._get_3d_backend() == 'mayavi'
        # the last thing we add is the Tube, so we need to go
        # vtkDataSource->Stripper->Tube->ModuleManager
        mod_man = fig.children[-1].children[0].children[0].children[0]
        title = mod_man.scalar_lut_manager.scalar_bar.title
    assert title == 'Connectivity'
```
I deduced sort of how to get to it from `mne/viz/backends/_pysurfer_mayavi.py` then looking at how `bgcolor` gets used there, and using `--pdb` then `interact` to mess around.

Reply:
(not 100% sure we should bother doing this, but it is one way to get it tested under Mayavi...)